implementations/cpp-jsoncons/bowtie_jsoncons.cpp

```diff
@@ -0,0 +1,129 @@
+      implementation["name"] = "jsoncons";
+      implementation["homepage"] = "https://danielaparker.github.io/jsoncons/";
+      implementation["issues"] =
+          "https://github.com/danielaparker/jsoncons/issues";
+      implementation["source"] = "https://github.com/danielaparker/jsoncons";
+
+      json dialects(json_array_arg,
+                    {"https://json-schema.org/draft/2020-12/schema",
+                     "https://json-schema.org/draft/2019-09/schema",
+                     "http://json-schema.org/draft-07/schema#",
+                     "http://json-schema.org/draft-06/schema#",
+                     "http://json-schema.org/draft-04/schema#"});
+      implementation["dialects"] = std::move(dialects);
+
+      response["implementation"] = std::move(implementation);
+
+      std::cout << print(response) << std::endl;
+    } else if (command == "dialect") {
+      // Set the dialect and respond
+      assert(started);
+      assert(message.contains("dialect") && message.at("dialect").is_string());
+      default_dialect = message.at("dialect").as<std::string>();
+
+      json response;
+      response["ok"] = true;
+      std::cout << print(response) << std::endl;
+    } else if (command == "run") {
+      assert(started);
+      assert(message.contains("seq"));
+      assert(message.contains("case") && message.at("case").is_object());
+      assert(message.at("case").contains("schema"));
+
+      const auto schema = message.at("case").at("schema");
+
+      try {
+        // Compile the schema and validate test cases
+        const auto compiled = jsonschema::make_json_schema(schema);
+        assert(message.at("case").contains("tests") &&
+               message.at("case").at("tests").is_array());
+
+        if (message.at("case").contains("registry")) {
+          // TODO implement registry
+        }
+
+        json response;
+        response["seq"] = message.at("seq");
+        response["results"] = json(json_array_arg);
+
+        for (const auto &test : message.at("case").at("tests").array_range()) {
+          assert(test.contains("instance"));
+
+          // Validate this test case and skip error messages
+          auto valid = true;
+          auto reporter =
+              [&valid](const jsonschema::validation_message &message)
+              -> jsonschema::walk_result {
+            valid = false;
+            // TODO Capture error messages
+            return jsonschema::walk_result::abort;
+          };
+          compiled.validate(test.at("instance"), reporter);
+
+          json test_result;
+          test_result["valid"] = std::move(valid);
+          response["results"].push_back(std::move(test_result));
+        }
+
+        std::cout << print(response) << std::endl;
+      } catch (const std::exception &error) {
+        // Report any errors during compilation or validation
+        json response;
+        response["errored"] = true;
```

Comment:
You're missing including `seq` here (which is the CI error now).

Reply:
Yes, I had fixed that and pushed before but I think it got overwritten when you force pushed your change earlier. I added that back.